modules/api/lambda_code/package/index.py

```diff
@@ -20,43 +43,90 @@ def lambda_handler(event, context):
-           }
+        #Filter out the rows where the given situation is "false" (ex : user_individual = false)      
+        filtered_results = []
+        for item in results:
+          if not any(value is False for value in item):
+            filtered_results.append(item)
+            
+        results_dict = {'niveau-alerte':'niveau'}
+        #Add all the restriction types into a list in order to decode them in utf-8
+        restriction_encoding = ["Réduction de prélèvement"] #TODO : find a way to query the rows with special characters
         
-        results_json["niveau-alerte"] = results[1]
-        #results_dict = results.pop(0)
-        #del results[0:2]
-        #results_json.update(results_dict)
+        #for restriction in restriction_level : 
+          #base64.b64encode(restriction).decode('utf-8')
+        
+        #Append the dictionary with reordered values in order to transform it in a json file
+        for item in filtered_results:
+          results_dict["niveau-alerte"] = item[1]
+          
+          if item[2] == "Sensibilisation":
+            results_dict["sensibilisation"] = item[2] #to change when the json model is updated
+            continue
+            
+          #elif item[2] == restriction_encoding[1]:
+            #results_dict["reduction-prelevement"] = item[2] #to change when the json model is updated
+            #continue
+          
+          elif item[2] == "Interdiction sauf exception":
+            results_dict["interdiction-sauf-exception"] = item[2] #to change when the json model is updated
+            continue
+            
+          elif item[2] == "Interdiction sur plage horaire":
+            results_dict["interdiction-plage-horaire"] = item[2] #to change when the json model is updated
+            continue
+            
+          elif item[2] == "Interdiction":
+            if item[4] == "Arrosage":
+              results_dict["interdiction"] = {
+                "arrosage-jardins-potagers": {
+                  "thematique": "arrosage",
+                  "libelle-personnalise":"interdiction d'arroser les jardins potagers",
+                  "en-savoir-plus":"Interdiction d'arroser les jardins potagers."}
+              } #to change when the json model is updated
+              continue
+            continue
+            
+          else:
+            break
+
+        #print ("Writing to file")
 
         # Create a json file with the results from the DB query
         #json_file = open("restrictions.json", "w")
         #json.dump(results_json, json_file, indent = 6)
         #json_file.close()
 
         print ("Sending results")
-        
-        return results
+
+        return results_dict # TODO : correct encoding of the responses (the special characters like "é" don't show well)
       
-      except Exception as exeption:
-        print("Error: ", exeption)
+      except OperationalError as err:
+        # pass exception to function
+        print_psycopg2_exception(err)
```

Comment:
```suggestion
      except Exception as e:
            logging.error('Failed to fetch restrictions. User: %s, Longitude: %s, Latitude: %s, Exception: %s', user, longitude, latitude, str(e))
```

Vaut mieux utiliser le logger c'est mieux exploitable.